package.json

```diff
@@ -20,19 +20,16 @@
     "slick-carousel": "^1.8.1"
   },
   "devDependencies": {
-    "@webdevstudios/js-coding-standards": "^1.0.0-beta1",
-    "@wordpress/eslint-plugin": "^4.0.0",
+    "@webdevstudios/js-coding-standards": "~1.0.1",
```

Comment:
You don't need to specify eslint stuff, js-coding-standards takes care of that stuff. Plus this version is super-duper newer than what we are using for coding standards (but I will eventually update).